infra/auth-logic/src/main/java/com/simprints/infra/authlogic/authenticator/ProjectAuthenticator.kt

```diff
@@ -88,4 +91,8 @@ internal class ProjectAuthenticator @Inject constructor(
             configManager.getPrivacyNotice(projectId, language).collect()
         }
     }
+
+    private suspend fun fetchFirmwareIfNeeded() {
```

Comment:
A private method for a single line feels a bit redundant. 

Also, what is the "ifNeeded" part referring to? There are no checks in the method.

Reply:
Wanted to make it clear that it's safe to call for all project configurations. Obviously not successful :D Will replace with a comment.